style.scss

```diff
@@ -0,0 +1,205 @@
+@mixin reset {
+  margin: 0;
+  padding: 0;
+}
+
+@mixin flexProperties {
+  display: flex;
+  justify-content: space-between;
+  flex-direction: row;
+  align-items: center;
+}
+
+@mixin circle ($width, $color) {
+  width: $width;
+  height: $width;
+  background: $color;
+  border-radius: $width/2;
+}
+
+$text-color: #FFFFFF;
+
+* {
+  @include reset;
```

Comment:
If we’re using reset in here only should we create the mixin for it? Shouldn’t we just use properties